tracer/test/Datadog.Trace.SourceGenerators.Tests/Datadog.Trace.SourceGenerators.Tests.csproj

```diff
@@ -7,10 +7,12 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.8.0" PrivateAssets="all" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
   </ItemGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\..\src\Datadog.Trace.SourceGenerators\Datadog.Trace.SourceGenerators.csproj" />
+    <ProjectReference Include="..\Datadog.Trace.TestHelpers\Datadog.Trace.TestHelpers.csproj" />
```

Comment:
Do we _need_ to add these? I hope not 🙁

Reply:
It's for getting the path of the solution. I guess it does not hurt...

Reply:
I misread this as being the source generator post itself 😅